nuxeo-java-client-test/src/test/java/org/nuxeo/client/ITDirectory.java

```diff
@@ -100,4 +102,45 @@ public void itCanCreateUpdateFetchDeleteDirectory() {
         directory.deleteEntry(result.getId());
     }
 
+    @Test
+    public void itCanSetOrderingPropertyInteger() {
+        Directory directory = nuxeoClient.directoryManager().directory("continent");
+
+        DirectoryEntry entryWithInteger = new DirectoryEntry();
+        entryWithInteger.setDirectoryName("continent");
+        entryWithInteger.putIdProperty("test1");
+        entryWithInteger.putLabelProperty("test1");
+        entryWithInteger.putObsoleteProperty(0);
+
+        entryWithInteger.putOrderingProperty(Integer.valueOf(Integer.MAX_VALUE).longValue());
+        DirectoryEntry resultWithInteger = directory.createEntry(entryWithInteger);
+        assertNotNull(resultWithInteger);
+
+        resultWithInteger = resultWithInteger.update();
+        assertEquals(Long.valueOf(Integer.MAX_VALUE), resultWithInteger.getOrderingProperty());
+
+        resultWithInteger.putOrderingProperty(Long.valueOf(Integer.MAX_VALUE));
+        assertEquals(Long.valueOf(Integer.MAX_VALUE), resultWithInteger.getOrderingProperty());
+        resultWithInteger = directory.updateEntry(resultWithInteger);
+        assertEquals(Long.valueOf(Long.MAX_VALUE), resultWithInteger.getOrderingProperty());
+    }
+
+    @Test
+    public void itCanSetOrderingPropertyLong() {
+        Directory directory = nuxeoClient.directoryManager().directory("continent");
+
+        DirectoryEntry entryWithLong = new DirectoryEntry();
+        entryWithLong.setDirectoryName("continent");
+        entryWithLong.putIdProperty("test2");
+        entryWithLong.putLabelProperty("test2");
+        entryWithLong.putObsoleteProperty(1);
+
+        entryWithLong.putOrderingProperty(Long.MAX_VALUE);
+        DirectoryEntry resultWithLong = directory.createEntry(entryWithLong);
+        assertNotNull(resultWithLong);
+
+        resultWithLong = resultWithLong.update();
+        assertEquals(Long.valueOf(Long.MAX_VALUE), resultWithLong.getOrderingProperty());
+    }
```

Comment:
`itCanSetOrderingPropertyLong` should be updated to look like the integer test, but with long.